doc/stake-pool-operations/query_stakepool.md

```diff
@@ -0,0 +1,107 @@
+# Querying a Stake Pool
+
```

Comment:
We should add something to the effect of "this command if for debugging purposes only and is not guaranteed to succeed due to the size of the ledger state"

Reply:
Done!